tests/test_messaging.py

```diff
@@ -547,7 +547,10 @@ def test_android_notification(self):
                     click_action='ca', title_loc_key='tlk', body_loc_key='blk',
                     title_loc_args=['t1', 't2'], body_loc_args=['b1', 'b2'], channel_id='c',
                     ticker='ticker', sticky=True,
-                    event_timestamp=datetime.datetime(2019, 10, 20, 15, 12, 23, 123),
+                    event_timestamp=datetime.datetime(
```

Comment:
Need a test case that triggers the no-timezone case as well.

Reply:
With the new changes do we still need a test case for a no-timezone? No time-zone cases will depend on the local timezone of the test servers

Reply:
I think one separate test case just focusing on `event_time` would be useful. Something like:

```
now = datetime.datetime.now()
expected = now.astimezone(utc).strftime(....)
# serialize test message with 'now' and compare against expected.
```